src/vapid-helper.js

```diff
@@ -6,6 +6,12 @@ const asn1 = require('asn1.js');
 const jws = require('jws');
 const url = require('url');
 
+/**
+ * DEFAULT_EXPIRATION is set to seconds in 12 hours
+ */
+const DEFAULT_EXPIRATION_SECONDS = 12 * 60 * 60;
```

Comment:
@gauntface Yup! FCM should I looked into the code of autopush and it does that too. It's standard spec behavior.